src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -1,5 +1,9 @@
 package seedu.addressbook.commands;
 
+import seedu.addressbook.ui.TextUi;
+
+import java.util.Scanner;
+
 /**
  * Clears the address book.
```

Comment:
Please consider updating the comments to indicate the new behavior that you have incorporated in this class.